programs/sbf/rust/custom_heap/src/lib.rs

```diff
@@ -13,7 +13,6 @@ use {
         alloc::{alloc, Layout},
         mem::{align_of, size_of},
         ptr::null_mut,
-        usize,
```

Comment:
Is this related / strictly necessary ? A little different then the rest of the changes in this PR